reader.go

```diff
@@ -111,6 +111,7 @@ type ISP struct {
 // using the Open function.
 type Reader struct {
 	mmdbReader *maxminddb.Reader
+	metadata   *maxminddb.Metadata
```

Comment:
We don't need to store the metadata struct here anymore. `Metadata()` can access it from `mmdbReader` directly.